design/user_type_test.go

```diff
@@ -0,0 +1,44 @@
+package design
+
+import "testing"
+
+func TestUserTypeExprIsCompatible(t *testing.T) {
+	var (
+		b = true
+		i = 1
+	)
+	cases := map[string]struct {
+		typ      DataType
+		values   []interface{}
+		expected bool
+	}{
+		"compatible": {
+			typ:      Int,
+			values:   []interface{}{i},
+			expected: true,
+		},
+		"not comatible": {
```

Comment:
typo

Reply:
I fixed it. https://github.com/goadesign/goa/pull/1622/commits/a2809739d435deac4ab5b2b1cabd0db7bf2c421b